### PR TITLE
DataTable add background color when sorted

### PR DESCRIPTION
**Pull request by halocline**
#### What does this PR do?

DataTable add background color when sorted

Closes https://github.com/grommet/hpe-design-system/issues/1241

#### Where should the reviewer start?

aries.js
[Table](https://deploy-preview-1243--keen-mayer-a86c8b.netlify.app/components/table)

#### What testing has been done on this PR?

local build

#### How should this be manually tested?

[preview build](https://deploy-preview-1243--keen-mayer-a86c8b.netlify.app/components/table)

#### What are the relevant issues?

https://github.com/grommet/hpe-design-system/issues/1241

#### Screenshots (if appropriate)

Screen Shot 2020-11-06 at 9 46 35 AM

#### Is this change backwards compatible or is it a breaking change?

Backwards

### CODE REVIEW

**Comment by halocline** [comment]
@vavalos5 Also note that the 'text-strong' color is being applied to the sortable icon now as well. That fix went into Grommet yesterday.

File: aries-site/src/themes/aries.js [hunk 1]
```diff
@@ -30,8 +30,14 @@ export const aries = deepMerge(hpe, {
       // background: undefined,
       border: { side: 'bottom' },
       color: 'text-strong',
-      extend: ({ column, sort, sortable }) => {
+      extend: ({ column, sort, sortable, theme }) => {
         return `
+          ${sort && sort.property === column && `
+            background: ${
+              theme.global.colors['background-contrast'][
```
Comment (taysea) [inline aries-site/src/themes/aries.js:37]: Nit picky but should we use `active-background` (same hex value, but feels like this is an "active" button)

**Review by taysea — COMMENTED** [review]
Just one small question.

File: aries-site/src/themes/aries.js (hunk 1 above)
Comment (halocline) [reply aries-site/src/themes/aries.js:37]: Good call. Will do.

**Review by vavalos5 — APPROVED** [review]
Perfecto!

**Review by vavalos5 — APPROVED** [review]

**Review by taysea — APPROVED** [review]
LGTM, just fix build.

**Pull request merged**